### DISCUSSION TITLE
Which is the current version of the foodsoft?

### DISCUSSION THREAD

**Discussion by fspade** [foodcoops/foodsoft#900] [General]
We are running foodsoft on a local server, but it says »Version: unknown«.
How can I as a user find out, which version we are running? I have no direct access to the source code ...
I like to find out if our local software needs an update.
Thanks in advance,
Frank

**Comment by JuliusR** [comment]
In all foodsoft installations I have seen, the version is shown on the right at the very bottom. You can have a look at the [demo instance of the global hosting for an example](https://app.foodcoops.net/demo/).

Are you sure that you are using the same "foodsoft"? Your local installation might be running a [completely different software with the same name](https://github.com/foodsoft/foodsoft).

**Comment by fspade** [comment]
Thanks for your reply.
Good question. Have a look for yourself: https://nahrungskette.net/foodsoft/
When I login the footer says:
aktueller Server: | Version: unknown | 2021-12-05 21:22:24
which looks strange to me, as if somebody tries to suppress that information.

**Reply by JuliusR** [reply]
The software behind your link is completely unrelated to the "foodsoft" of this repository. It is a completely different software, but it (unfortunately) has the same name. You can try to contact experts of that other software [here](https://github.com/foodsoft/foodsoft). If that does not work, I can offer to connect you with someone who should be able to answer your questions. That will be private communication, just tell me if you wish.

**Comment by JuliusR** [comment]
To answer the title question: As of December 5th, 2021, the current version of this foodsoft (foodcoops/foodsoft) is **v4.7.1** (see [release notes](https://foodcoops.net/2021/01/17/foodsoft-4-7-1/), [GitHub tag](https://github.com/foodcoops/foodsoft/tree/v4.7.1), [Docker Hub Tags](https://registry.hub.docker.com/r/foodcoops/foodsoft/tags)). It was released about a year ago.

**Comment by fspade** [comment]
Thank you. The question should have been: Which is the current version of **our** foodsoft?
I posted that question in the other software's forum. Thanks.

**Comment by RayOei** [comment]
No need to keep this active.
